### PR TITLE
Summary value fix

### PR DESCRIPTION
**Pull request by anandtiwary**

### CODE REVIEW

File: projects/observability/src/shared/components/timeline-card-list/timeline-card-list.component.ts [hunk 1]
```diff
@@ -49,6 +50,9 @@ export class TimelineCardListComponent implements AfterContentInit {
   @Input()
   public selectedIndex?: number;
 
+  @Input()
+  public hideTimeView?: boolean = false;
```
Comment (aaron-steinfeld) [inline projects/observability/src/shared/components/timeline-card-list/timeline-card-list.component.ts:54]: could we flip this to something like showTime? I think it's easier to mentally parse boolean properties that are in the affirmative.

File: projects/observability/src/shared/components/timeline-card-list/timeline-card-list.component.scss [hunk 2]
```diff
@@ -42,24 +45,22 @@
       display: flex;
       flex-direction: row;
       width: 100%;
-      border: 1px solid $gray-2;
+      //border: 1px solid $gray-2;
```
Comment (arjunlalb) [inline projects/observability/src/shared/components/timeline-card-list/timeline-card-list.component.scss:48]: remove this if not needed?

File: projects/components/src/menu-dropdown/menu-dropdown.component.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ import { MenuItemComponent } from './menu-item/menu-item.component';
   styleUrls: ['./menu-dropdown.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <div class="menu-dropdown">
+    <ht-event-blocker event="click" class="menu-container" [enabled]="this.disabled">
```
Comment (arjunlalb) [inline projects/components/src/menu-dropdown/menu-dropdown.component.ts:11]: now both parent and child elements have `menu-container` class on them. The parent should be `menu-dropdown`

File: projects/observability/src/shared/components/timeline-card-list/timeline-card-list.component.ts (hunk 1 above)
Comment (anandtiwary) [reply projects/observability/src/shared/components/timeline-card-list/timeline-card-list.component.ts]: Sure

Comment (anandtiwary) [reply projects/observability/src/shared/components/timeline-card-list/timeline-card-list.component.ts]: Addressed in the fixing-timeline-card-view PR

File: projects/observability/src/shared/components/timeline-card-list/timeline-card-list.component.scss (hunk 2 above)
Comment (anandtiwary) [reply projects/observability/src/shared/components/timeline-card-list/timeline-card-list.component.scss]: Addressed in the fixing-timeline-card-view PR

**Review by jake-bassett — APPROVED** [review]

**Pull request merged**